deeplite/tflite_profiler/flops/compute_flops.py

```diff
@@ -0,0 +1,77 @@
+import tensorflow as tf
+import tflite
+import numpy as np
+from deeplite.profiler.utils import Device
+
+from deeplite.profiler.metrics import Flops
+from deeplite.profiler import ProfilerFunction
+
+
+class ComputeFlops(ProfilerFunction):
+    def get_bounded_status_keys(self):
+        return Flops()
+
+    def __call__(self, model, data_splits, batch_size=1, device=Device.CPU, include_weights=True):
+        temp_model = tflite.Model.GetRootAsModel(model, 0)
+
+        with tf.device('gpu' if device == Device.GPU else 'cpu'):
+            return self._compute_flops(temp_model)
+
+    def _compute_flops(self, model):
+        subgraph = model.Subgraphs(0)
```

Comment:
this method has to be test covered properly

Reply:
done